sdk/eventhub/Azure.Messaging.EventHubs/src/Amqp/AmqpConsumer.cs

```diff
@@ -401,12 +402,12 @@ public override async Task CloseAsync(CancellationToken cancellationToken)
                 link = await ConnectionScope.OpenConsumerLinkAsync(
                     consumerGroup,
                     partitionId,
-                    CurrentEventPosition,
+                    eventStartingPosition,
```

Comment:
If you expand the context of the diff here, you'll see we were just dropping these two arguments on the floor.  The only call site was passing in `CurrentEventPosition` and `CancellationToken.None` for `eventStartingPosition` and `cancellationToken`, so this should result in a change in practice.

Reply:
Yeah, this was a recent refactor and it looks like I fixed the token in a recent commit but overlooked the starting position.   Agreed that this is correct and not behavior changing.